src/players/Player.js

```diff
@@ -0,0 +1,47 @@
+import { MESSAGE } from '../constants/constants.js';
+import View from '../view/View.js';
+
+export default class Player {
+
+  async inputNumber() {
+    const userInput = await View.readInput(MESSAGE.input);
+
+    if (Player.isValidNumber(userInput)) {
+      this._playerNumber = userInput.split('').map(num => parseInt(num, 10));
```

Comment:
매핑할 때 `Array.from()`을 사용하면 중간 배열 생성을 방지할 수 있어요!

```suggestion
      this._playerNumber = Array.from(userInput, Number);
```

[참고 자료]
https://github.com/parksb/javascript-style-guide#arrays--mapping

Reply:
아직 메소드 사용이 익숙치 않네요.
알려주셔서 감사합니다!